src/freenet/node/PeerManager.java

```diff
@@ -238,58 +238,60 @@ private boolean readPeers(File peersFile, NodeCrypto crypto, OpennetManager open
 		BufferedReader br = new BufferedReader(ris);
 		File brokenPeersFile = new File(peersFile.getPath() + ".broken");
 		DroppedOldPeersUserAlert droppedOldPeers = new DroppedOldPeersUserAlert(brokenPeersFile);
-		try { // FIXME: no better way?
-			while(true) {
-				// Read a single NodePeer
-				SimpleFieldSet fs;
-				fs = new SimpleFieldSet(br, false, true);
-				try {
-					PeerNode pn = PeerNode.create(fs, node, crypto, opennet, this);
-					if(oldOpennetPeers) {
-					    if(!(pn instanceof OpennetPeerNode))
-					        Logger.error(this, "Darknet node in old opennet peers?!: "+pn);
-					    else
-					        opennet.addOldOpennetNode((OpennetPeerNode)pn);
-					} else
-						addPeer(pn, true, false);
-				} catch(FSParseException e2) {
-					Logger.error(this, "Could not parse peer: " + e2 + '\n' + fs.toString(), e2);
-					System.err.println("Cannot parse a friend from the peers file: "+e2);
-					someBroken = true;
-					continue;
-				} catch(PeerParseException e2) {
-					Logger.error(this, "Could not parse peer: " + e2 + '\n' + fs.toString(), e2);
-					System.err.println("Cannot parse a friend from the peers file: "+e2);
-					someBroken = true;
-					continue;
-				} catch(ReferenceSignatureVerificationException e2) {
-					Logger.error(this, "Could not parse peer: " + e2 + '\n' + fs.toString(), e2);
-					System.err.println("Cannot parse a friend from the peers file: "+e2);
-					someBroken = true;
-					continue;
-				} catch (RuntimeException e2) {
-					Logger.error(this, "Could not parse peer: " + e2 + '\n' + fs.toString(), e2);
-					System.err.println("Cannot parse a friend from the peers file: "+e2);
-					someBroken = true;
-					continue;
-					// FIXME tell the user???
-				} catch (PeerTooOldException e) {
-				    if(crypto.isOpennet()) {
-				        // Ignore.
-				        Logger.error(this, "Dropping too-old opennet peer");
-				    } else {
-				        // A lot more noisy!
-				        droppedOldPeers.add(e, fs.get("myName"));
-				    }
-                    someBroken = true;
-                    continue;
-                }
+		List<SimpleFieldSet> peerEntries = new ArrayList<>();
+		// read the peers file
+		try {
+			while (true) {
+			  peerEntries.add(new SimpleFieldSet(br, false, true));
```

Comment:
You’re mixing different kinds of whitespace here.

Reply:
_resisting the urge to do a full file whitespace cleanup that would cause merging issues_